transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py

```diff
@@ -344,7 +275,8 @@ def geocode_address(address):
     "Greece",
     "Kazakhstan",
     "Kyrgyzstan",
-    "Kosovo (UN SC resolution 1244)",
```

Comment:
I think this is the name proposed by our colleagues @lbassil

Reply:
Well I was trying to troubleshoot the exception for Kosovo, the data exported from the SDMX use the name Kosovo and not as we have in out init file. In any event, I also remove this change from this poverty branch. But keep it in mind please.

Reply:
you're right @lbassil, SDMX label of countries don't have the UN resolution thing nor the UK left EU ... legends, well raised! We could add these legends as long as they are used only for dash display but not to match data right?

Reply:
Hi, well when you call the geocode function geocode_address and you pass the country name, it doesn't find the one we use for Kosovo... that's why it is returning null for the map lat and long...